### PR TITLE
fix: example labels

### PR DESCRIPTION
**Pull request by Andras-Csanyi**
## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [x] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## Does this PR introduce a breaking change?    
- [ ] Yes
- [x] No

### CODE REVIEW

File: examples/enterprise-management.v1.test.js [hunk 1]
```diff
@@ -107,7 +107,8 @@ describe('EnterpriseManagementV1', () => {
     });
 
     originalLog('createAccountGroup() result:');
-
+  
+    originalLog('() result:');
```
Comment (padamstx) [inline examples/enterprise-management.v1.test.js:111]:
```suggestion
    originalLog('createAccountGroup() result:');
```

File: examples/global-catalog.v1.test.js [hunk 2]
```diff
@@ -353,10 +359,10 @@ describe('GlobalCatalogV1', () => {
 
     globalCatalogService.updateVisibility(params)
       .then(res => {
-        console.log('updateVisibility() response status code: ' + res.status);
+        done();
       })
       .catch(err => {
-        console.log('updateVisibility() returned the following error: ' + err);
+        console.warn('updateVisibility() returned the following error: ' + err);
```
Comment (padamstx) [inline examples/global-catalog.v1.test.js:365]:
```suggestion
        console.warn(err);
```

**Review by padamstx — APPROVED** [review]
Just a couple of minor changes needed.  I'll approve now assuming those are addressed.

**Review by pyrooka — APPROVED** [review]

**Pull request merged**